emission/analysis/modelling/tour_model/tour_model_matrix.py

```diff
@@ -0,0 +1,275 @@
+# Our imports
+import emission.simulation.markov_model_counter as esmmc
+from emission.core.our_geocoder import Geocoder
+
+# Standard imports
+import numpy as np
+import math, datetime, heapq
+import networkx as nx
+import matplotlib.pyplot as plt
+
+# Constants
+DAYS_IN_WEEK = 7
+HOURS_IN_DAY = 24
+MODES_TO_NUMBERS = {"walking" : 0, "car" : 1, "train" : 2, "bart" : 3, "bike" : 4}
+NUM_MODES = len(MODES_TO_NUMBERS)
+
+class Commute(object):
```

Comment:
Nit: I am not sure I like the "Commute" terminology. Not all trips are commutes. If you go to cheeseboard to get pizza, that is a trip. Is that a commute?